docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -35,112 +32,122 @@ documented on the identity providers website. Here are a few links:
 - [Google Identity Platform](https://developers.google.com/identity/protocols/OpenIDConnect)
 - [Keycloak Client Registration](https://www.keycloak.org/docs/latest/securing_apps/index.html#\_client_registration)
 
-Add your OIDC connector information to `teleport.yaml`. A few examples are
-provided below.
-
 <Admonition type="tip">
```

Comment:
Should we add a title "Google Workspace" to this admonition?

As a reader I often find titles helpful because it quickly let's me know whether I can skip over the admonition or whether it applies to me and I should fully understand it.

Reply:
```suggestion
<Admonition type="tip" title="Google Workspace">
```